src/pages/api/modules/usenet/index.ts

```diff
@@ -0,0 +1,70 @@
+import { getCookie } from 'cookies-next';
+import dayjs from 'dayjs';
+import duration from 'dayjs/plugin/duration';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { Client } from 'sabnzbd-api';
+import { getConfig } from '../../../../tools/getConfig';
+import { getServiceById } from '../../../../tools/hooks/useGetServiceByType';
+import { Config } from '../../../../tools/types';
+
+dayjs.extend(duration);
+
+export interface UsenetInfoRequestParams {
+  serviceId: string;
```

Comment:
This is pretty much not needed as we can just find the service from the API. (See Overseerr module for example) 
This way is fine

Reply:
For right now you can only retrieve data for a single service since pagination across multiple services is quite a hassle.